public/app/core/components/Tooltip/withPopper.tsx

```diff
@@ -9,13 +9,15 @@ export interface UsingPopperProps {
   content: string | ((props: any) => JSX.Element);
   className?: string;
   refClassName?: string;
+  theme?: string;
```

Comment:
What do you think about specifying the type more precisely? `type TooltipTheme: 'default' | 'error' ...`

Reply:
Absolutely!